lib/pure/terminal.nim

```diff
@@ -17,6 +17,16 @@
 ## ``showCursor`` before quitting.
 
 import macros
+from strutils import toLowerAscii
+import tables
+import colors
+export colors
```

Comment:
I don't think ``terminal`` should export ``colors``.

Reply:
Done.